private/constellation/constellation.go

```diff
@@ -47,19 +46,23 @@ func New(path string) (*Constellation, error) {
 	}
 	// We accept either the socket or a configuration file that points to
 	// a socket.
-	isSocket := info.Mode() & os.ModeSocket != 0
+	isSocket := info.Mode()&os.ModeSocket != 0
```

Comment:
Please do ensure to have the spaces in place `isSocket := info.Mode() & os.ModeSocket != 0`
@conor10

Reply:
Thanks @Puneetha17 - @atoulme that's the only feedback from us, otherwise looks good 👍 (build appears to be failing however...)

Reply:
Thanks folks. I am going to take a look at that failing build.